vue-frontend/src/components/portfolio/VideoSection.vue

```diff
@@ -135,7 +135,7 @@
   </section>
 
   <!-- large screen slider -->
-  <section v-else-if="response.slider" class="tw-relative">
+  <section v-else-if="response.slider" class="tw-relative tw-mt-20 lg:tw-mt-0">
```

Comment:
The bottom space is too much, can't we reduce `py` of the branding section?

Reply:
No need to add or remove the padding in this section. space is due to branding section, fix their